resources/views/tickets/index.blade.php

```diff
@@ -6,7 +6,23 @@
 
     <div class="m4">
         <a class="button " href="{{ route("tickets.create") }}">@icon(plus) {{ __('ticket.newTicket') }}</a>
-        <a class="button secondary" id="mergeButton" onclick="onMergePressed()"> {{ __('ticket.merge') }}</a>
+        <a class="button secondary" id="mergeButton" title="{{__('ticket.mergeDesc')}}" onclick="mergeTickets()"> {{ __('ticket.merge') }}</a>
+        <div class="dropdown-wrapper">
```

Comment:
I believe there is already a dropdown and dropdown-container classes that do the dropdown automatically

Reply:
Maybe we can put all this dropwon into a view component so it is easier to read on index.blade.php

Reply:
`.dropdown` is the button on which you click
`.dropdown-container` has the buttons that should pop up, this is positioned absolutely
`.dropdown-wrapper` is here to give a `position: relative` around these 2 guys, otherwhise `.dropdown-container` ends up in a completely different place

Reply:
I see, umm maybe we can fix the dropdown-container instead then

Reply:
Well it makes sense that it is positioned absolute basically, but I haven't found any documentation on the Hurricane framework that you are using